source/s3_meta_request.c

```diff
@@ -327,12 +320,14 @@ static void s_s3_request_on_shutdown(void *user_data) {
  * file-based python input stream for reporting the progress
  */
 struct aws_input_py_stream_file_impl {
+    struct aws_input_stream base;
+    struct aws_allocator *allocator;
```

Comment:
don't need to store the allocator. in aws-crt-python there's only one global allocator